D2L_Users_Enrollments/queries_root/c08_e_g.named_queries.xml

```diff
@@ -15,78 +15,72 @@
+                'Parent'                    AS "role_name",
 
-                cs_cc.schoolid_cc.course_number_cc.termid_cc.expression_cc.sectionid
-                code: cs_3_SPHYIBS2_3200_D_28636
-                
-                NOTE: X block is added as a place-holder for courses that do not appear on the 
-                schedule such as MS Enrichment, self study IB courses SSST
-                */
-                'cs_'||cc.schoolid||'_'||cc.course_number||'_'||cc.TermID||'_'||DECODE(regexp_replace(cc.expression, '[^0-9]', ''), 
-                1, 'A', 
-                2, 'B', 
-                3, 'C', 
-                4, 'D', 
-                5, 'E', 
-                6, 'F', 
-                7, 'G', 
-                8, 'H', 
-                9, 'ADV',
-                10, 'X',
-                'UNKNOWN')||'_'||cc.sectionid as "parent_code"
-            from GUARDIAN GUARDIAN,
-                COURSES COURSES,
-                SECTIONS SECTIONS,
-                STUDENTS STUDENTS,
-                CC CC,
-                GUARDIANSTUDENT GUARDIANSTUDENT,
-                TERMS TERMS
-            where GUARDIANSTUDENT.STUDENTSDCID=STUDENTS.DCID
-                and STUDENTS.ID=CC.STUDENTID
-                and CC.SECTIONID=SECTIONS.ID
-                and TERMS.id = cc.termid
-                and SECTIONS.COURSE_NUMBER=COURSES.COURSE_NUMBER
-                and GUARDIANSTUDENT.GUARDIANID=GUARDIAN.GUARDIANID
-                and STUDENTS.ENROLL_STATUS =0
-                and STUDENTS.GRADE_LEVEL >=5
                 /*
-                Explicitly only include guardianstudent.guardianrelationshiptypeid
-                of type 10, 20.
-                
-                As of Jan 2023 we can only reliably create parent accounts that appear on the 
-                Admissions Screen in PowerSchool SIS under the mother/father contact details.
-                
-                It may be possible to improve this when moving to the PS SIS Contacts feature
+                 * use the following format to ensure that section codes are unique
+                 * and remain constant even if the teacher is reassigned
+                 * 
+                 * cs_cc.schoolid_cc.course_number_cc.termid_cc.expression_cc.sectionid
+                 * code: cs_3_SPHYIBS2_3200_D_28636
+                 * 
+                 * NOTE: X block is added AS a place-holder for courses that do not appear on the 
+                 * schedule such AS MS Enrichment, self study IB courses SSST
+                 */
+                'cs_' || cc.schoolid || '_' || cc.course_number || '_' || cc.termid || '_' || DECODE(regexp_replace(cc.expression, '[^0-9]', ''), 
+                    1, 'A', 
+                    2, 'B', 
+                    3, 'C', 
+                    4, 'D', 
+                    5, 'E', 
+                    6, 'F', 
+                    7, 'G', 
+                    8, 'H', 
+                    9, 'ADV',
+                    10, 'X',
```

Comment:
need to swap these everywhere. PowerSchool apparently does not index these values, but rather keeps them as a sorted list. PS sorts these A..Z, AA. This is unexpected, but managable.